ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/editor/texteditor/TextEditorPresenter.java

```diff
@@ -292,6 +293,7 @@ public void onDocumentReceived(final String content) {
                     @Override
                     public void onContentInitialized() {
                         document.setCursorPosition(currentCursor);
+                        generalEventBus.fireEvent(new FileContentUpdatedEvent(document.getFile()));
```

Comment:
When debugger open external jar file, editor can be without source. And user click "Download source". We need FileContentUpdatedEvent for render debug line when content was downloaded and updated.